shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -91,6 +92,14 @@ - (void)dispatchMouseEvent:(nonnull NSEvent*)event phase:(FlutterPointerPhase)ph
  */
 - (void)dispatchKeyEvent:(NSEvent*)event ofType:(NSString*)type;
 
+/**
+ * Forwarding texture copy request to the corresponding texture via |textureID|,
+ * |width| and |height| may be used to get a specific sized texture in the future.
```

Comment:
No need to comment on the meaning of width/height, since this is just forwarding. (Specifically it seems like this comment is talking about implementation details that are in a different class.)